web/src/main/java/org/akaza/openclinica/service/ImportServiceImpl.java

```diff
@@ -1306,6 +1358,38 @@ private Object validateItem(ImportItemDataBean itemDataBean, CrfBean crf, EventC
                 return new ErrorObj(NO_CHANGE, null);
 
             } else {
+            	if(isEventCrfCompleted(eventCrf)) {         		            		                
+                    
```

Comment:
You may want to create separate method to create queries when event crf has status completed. This method is used to validate Items and not creating queries. it will cluster this method.

Reply:
created a new create method per feedback